WordPress/Classes/ViewRelated/NUX/LoginViewController.swift

```diff
@@ -11,24 +13,76 @@ class LoginViewController: NUXAbstractViewController {
         return facade
     }()
 
+
+    // MARK: Lifecycle Methods
+
     override func viewDidLoad() {
         super.viewDidLoad()
         displayError(message: "")
         setupNavBarIcon()
+        styleInstructions()
 
         if let error = errorToPresent {
             displayRemoteError(error)
         }
     }
 
+
+    // MARK: - Setup and Configuration
+
     /// Places the WordPress logo in the navbar
     ///
     func setupNavBarIcon() {
-        let image = UIImage(named: "social-wordpress")
-        let imageView = UIImageView(image: image?.imageWithTintColor(UIColor.white))
+        let image = Gridicon.iconOfType(.mySites)
+        let imageView = UIImageView(image: image.imageWithTintColor(UIColor.white))
         navigationItem.titleView = imageView
     }
 
+    /// Configures instruction label font
+    ///
+    func styleInstructions() {
+        // iOS won't return UIFontWeightMedium for dynamic system font :(
+        // -So instead get the dynamic font size, then ask for the non-dynamic font at that size
+        let fontToGetSize = WPStyleGuide.fontForTextStyle(.subheadline)
+        instructionLabel?.font = UIFont.systemFont(ofSize: fontToGetSize.pointSize, weight: UIFontWeightMedium)
```

Comment:
This same trick is done in the prologue vc too.  Maybe factor into a WPStyleGuide extension for reuse?